benches/census90.rs

```diff
@@ -28,7 +28,7 @@ fn main() {
     let grouping = GroupingLabel::from("census90");
     let table = csv_to_json_table::<CensusEntry>(
         "dev_utils/src/data_models/data-raw/census90.txt",
-        &grouping.to_string(),
+        format!("{}", &grouping).as_str(),
```

Comment:
I see a lot of these `to_string()` being replaced with `format!("{}")`. Why? The current one is shorter.